app/views/bookmarks/_filters.html.erb

```diff
@@ -131,6 +131,12 @@
       <dd class="submit actions"><%= f.submit ts("Sort and Filter") %></dd>
     </dl>
 
+    <% if @pseud || @user || @collection || @tag %>
```

Comment:
You can use `@owner` here.

https://github.com/otwcode/otwarchive/blob/6551990716b90f441ce9cbbb476a42a6d4142185/app/controllers/bookmarks_controller.rb#L332

It includes `@bookmarkable` as well, but if there's a `@bookmarkable` then the filters won't be loaded anyway, so we're still good.